frontend/src/app/features/dashboard/components/main-dashboard/main-dashboard.component.html

```
@@ -51,15 +51,29 @@ <h1 class="no-library-header">
             @if (dashboardConfig(); as config) {
-              @for (scroller of config.scrollers; track scroller.id) {
+              @for (scroller of config.scrollers; track scroller.id; let idx = $index) {
                 @if (scroller.enabled) {
-                  <app-dashboard-scroller
-                    [bookListType]="scroller.type"
-                    [title]="scroller.title"
-                    [isMagicShelf]="scroller.type === ScrollerType.MAGIC_SHELF"
-                    [useSquareCovers]="scroller.type === ScrollerType.LAST_LISTENED"
-                    [books]="getBooksForScroller(scroller)">
-                  </app-dashboard-scroller>
+                  @if (idx < 2) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)